### ISSUE TITLE
Permalink not working with multiple git repos in workspace

### ISSUE THREAD

**Issue by mnblonsky** [hogashi/vscode-copy-github-permalink#17]
Hi,
I love this extension, but it doesn't work for some of my workspaces. It works fine when I only have 1 git repository in the workspace. But I get issues when there's multiple git repositories in the same workspace. The URL always starts with one of the repo's webpages, even if I'm requesting a permalink from the other repo. Is this something you can fix?

**Comment by hogashi** [comment]
@mnblonsky Thank you for using this extension and creating the issue! ⭐

>But I get issues when there's multiple git repositories in the same workspace.

Oh, I see.  I haven't expected that situation.  I'll take a look whether I can fix or not in a while!

**Comment by hogashi** [comment]
@mnblonsky I've just published v0.2.0 🎍 I hope it has been fixed!

**Comment by mnblonsky** [comment]
I'm getting an error: `copy-github-permalink can't get git repo`

I don't see any more details, although I'm not sure where to look. Looks like the extension was updated to v0.2.3

**Comment by hogashi** [comment]
oh sorry... I'll revert that

**Comment by mnblonsky** [comment]
Looks like it's for every repo, including single git repos

**Comment by mnblonsky** [comment]
I just tried v0.2 and 0.2.2, I get the same error

**Comment by hogashi** [comment]
Thanks! Ok, then I'll revert that anyway.

**Comment by hogashi** [comment]
#18 is now reverted in #33 and published as v0.2.4.  I'll take a look further.  Thank you for your report!

**Comment by FelixEnescu** [comment]
Still happens to me on 0.2.4
Any place I can look, any logs I can provide to help?

**Comment by hogashi** [comment]
Thank you @FelixEnescu .  I'm happy if there are some repositories to reproduce the error.

**Comment by FelixEnescu** [comment]
> Thank you @FelixEnescu . I'm happy if there are some repositories (and procedure) to reproduce the error.
> 
> * memo: procedure? https://code.visualstudio.com/docs/editor/multi-root-workspaces

I can reproduce with the folloiwng steps in a WSL terminal window:

```

mkdir test
cd test
git clone git@github.com:hogashi/vscode-copy-github-permalink.git
git clone git@github.com:hogashi/amp-autofix.git
code .
```

I get a window like this: 
![image](https://user-images.githubusercontent.com/2485129/202463062-1f74c147-4955-4dcf-9350-ca0c1312e42b.png)

I open a file from `amp-autofix.git` repo:
![image](https://user-images.githubusercontent.com/2485129/202463343-61642065-cd2d-424b-957c-cb2555f7b4a5.png)

and I get: https://github.com/hogashi/vscode-copy-github-permalink/blob/359b7f4970dbd71afa561ad06b313fe8c57c09a3#L18

Thank you for your amazing extension! It is very helpfull!

**Comment by hogashi** [comment]
Thank you!  I'll give it another try...

**Comment by hogashi** [comment]
Thanks for the reports, I've found that https://github.com/hogashi/vscode-copy-github-permalink/issues/17#issuecomment-1252596253 error in #18 was the Windows problem (maybe) and I can repoduce it.  That was the path format problem and I'm going to fix it... https://nodejs.org/api/path.html#windows-vs-posix

**Comment by hogashi** [comment]
I've just made a fix #34 and published as v0.2.5 .  Please check it out ☕

**Comment by FelixEnescu** [comment]
> I've just made a fix #34 and published as v0.2.5 . Please check it out ☕

All ok! Confirm works like a charm!

Thank you again for your great extension!

**Comment by mnblonsky** [comment]
Works for me too. Thank you!
